src/pages/search/index.tsx

```diff
@@ -465,6 +472,50 @@ const Search: InferGetServerSidePropsType<typeof getServerSideProps> = ({
                       )}
                     </SlideOut>
                   )}
+                  <SlideOut showCloseButton={false}>
+                    <div className="text-sm text-text-secondary flex flex-col gap-5">
+                      <Accordian
+                        title={getFilterLabel("Region", "region", router.query[QUERY_PARAMS.category], themeConfig)}
+                        data-cy="regions"
+                        startOpen
+                      >
+                        <InputListContainer>
+                          {regions.map((region) => (
+                            <InputCheck
+                              key={region.slug}
+                              label={region.display_value}
+                              checked={regionFilters && regionFilters.includes(region.slug)}
+                              onChange={() => {
+                                handleRegionChange(region.slug);
+                              }}
+                              name={`region-${region.slug}`}
+                            />
+                          ))}
+                        </InputListContainer>
+                      </Accordian>
+                      <Accordian
```

Comment:
I think we should add the quick search attribute to this so that we can use the filtering